server/models/Conversation.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+const Profile = require("./Profile");
+const Schema = mongoose.Schema;
+const message = new Schema({
+  sender: { type: Schema.Types.ObjectId, ref: "Profile" },
+  message: String,
+  date: Date,
+});
+
+const conversationSchema = new Schema({
+  requestId: {
```

Comment:
As per my understanding if we are creating a new conversation for each new request, won't there be duplicate conversations between users?